utils/tail/tail.go

```diff
@@ -104,6 +104,8 @@ func (t *Tail) attemptOpen() error {
 			t.reader = bufio.NewReader(t.file)
 			return nil
 		}
+		glog.Errorf("open log file %s error. %s", t.filename, err)
```

Comment:
Since this is in a retry loop, it should be logged at info, maybe `V(4)`.

I suggested saving the error to a variable outside the loop, and then including it in the returned error at line 119. e.g.
```
err := fmt.Errorf("can't open log file %s: %v", t.filename, lastErr)
```